GradientLoadingBar/Classes/GradientLoadingBarController.swift

```diff
@@ -100,7 +100,7 @@ open class GradientLoadingBarController {
             gradientActivityIndicatorView.heightAnchor.constraint(equalToConstant: height),
 
             gradientActivityIndicatorView.leadingAnchor.constraint(equalTo: superview.leadingAnchor),
-            gradientActivityIndicatorView.trailingAnchor.constraint(equalTo: superview.trailingAnchor)
+            gradientActivityIndicatorView.widthAnchor.constraint(equalToConstant: superview.frame.size.width)
```

Comment:
This won't work correctly in case the user rotates the device.
We would need to setup a constraint with a multiplier to the superview, so something like:
```swift
gradientActivityIndicatorView.widthAnchor.constraint(equalTo: superview.widthAnchor, multiplier: progress)
```